lib/imagery/iclass_perimeter.c

```diff
@@ -198,7 +198,7 @@ int make_perimeter(struct line_pnts *points, IClass_perimeter *perimeter,
     first = -1;
     prev = count - 1;
     for (i = 0; i < count; prev = i++) {
-        /* non absurd polygon has vertexes with different y coordinate */
+        /* non absurd polygon has vertices with different y coordinate */
```

Comment:
```suggestion
        /* non absurd polygon has vertices with different y coordinates */
```

Reply:
Apologies, just seeing this now.

Reply:
No problem, we absolutely needed to make the CI finish off during the night, as we had a queue of about 5-7 hours, so I really wanted to not have the correction in the following day, slowing us down uselessly.

I don't mind personally reviewing bigger PRs